Documentation/building/test-configuration.md

```diff
@@ -1,41 +1,54 @@
-## General Test Infrastructure Notes ##
+# General Test Infrastructure
 
-### Kinds of Build Properties ###
-* Build Only
-> `<CLRTestKind>BuildOnly</CLRTestKind>`
-
- * Builds an executable. 
- * Will not execute it. 
+## Test "Kind"
 
+* Build Only
+  * Builds an executable.
+  * Will not execute.
+  * e.g. `<CLRTestKind>BuildOnly</CLRTestKind>`
 * Run Only
-> `<CLRTestKind>RunOnly</CLRTestKind>`
+  * Can use Output of Build and Run Project with different command line arguments.
```

Comment:
If I'm interpreting this correctly, I think something like "Can use output of Build or BuildAndRun projects with different command line arguments." would be clearer.